relay-dynamic-config/src/defaults.rs

```diff
@@ -143,6 +147,66 @@ pub fn add_span_metrics(project_config: &mut ProjectConfig) {
                 })
                 .into(),
         },
+        MetricSpec {
+            category: DataCategory::Span,
+            mri: "c:spans/data.llm.llm_completion_tkens@none".into(),
+            field: Some("span.data.llm_completion_tkens".into()),
+            condition: Some(RuleCondition::eq("span.op", "ai.llm.completion")),
+            tags: [
+                ("span.sentry_tags.", "group"),
+                ("span.sentry_tags.", "description"),
```

Comment:
This looks like it contains the prompt and I would think it's high cardinality since it could contain user input. Is this correct?

If yes, we can't use the description or group as a tag.

Reply:
In general the way OpenAI recommends one to set up their prompts is to have the first prompt be a "system" prompt, which sets the behavior of the assistant. For example, in our own [use case,](https://github.com/getsentry/sentry/blob/c8071e2283a5b14bba9180dddc2fedc654177277/src/sentry/api/endpoints/event_ai_suggested_fix.py#L256) we only have 4 prompts, so the grouping should work. In basic use cases.

Misuses of the API yes would resust in high cardinality descriptions, but this shouldn't be the norm.

Reply:
As we also identified on Slack, you're going to hit truncation issues. I would not tag with the description, keep hashing the full description for the group though (so beware of https://github.com/getsentry/relay/pull/2749).

Reply:
I've gone ahead and removed description from the metrics tags here. Yes, I think for LMM prompts we'd want the group to be based off of the un-truncated description.